lib/kafo/execution_environment.rb

```diff
@@ -42,6 +42,13 @@ def configure_puppet(settings = {})
       PuppetConfigurer.new(puppet_conf, settings)
     end
 
+    def build_command(code, options: [], settings: {}, use_answers: false)
```

Comment:
I think I might like `build_puppet_command` to make it clearer to understand what sort of command is being built up. This method signature has the side affect of being able to be read as "build command" a noun rather than build as a verb.

Reply:
Alternative: would `command` make sense? You would get `execution_env.command(code)`.

Reply:
Can the command be anything I want? or is it specific to Puppet?